eo-runtime/src/main/eo/org/eolang/tuple.eo

```diff
@@ -57,10 +57,17 @@
         index.lt 0
         index.gte len
       error "Given index is out of tuple bounds"
-      if.
-        index.lt (len.plus -1)
-        ^.head.at index
-        ^.tail
+      ^.at-without-checks index
+
+  # Takes element from the tuple without checking i.
+  # i must be less than length and greater or equal 0.
+  # This is faster than at.
+  [i] > at-without-checks
```

Comment:
@levBagryansky without checking the source code it's not really clear why it's faster and what "without checks" means.
I would suggest not to extend `tuple` interface, but speed up `at` attribute by moving this `at-without-checks` inside `at`

Reply:
@levBagryansky so when you enter `at` you do all your checks just once and then recursively calls `at-without-checks`

Reply:
@maxonfjvipon let's rename it somehow

Reply:
@maxonfjvipon done